examples/dreambooth/train_dreambooth.py

```diff
@@ -353,14 +368,22 @@ def __init__(
         else:
             self.class_data_root = None
 
-        self.image_transforms = transforms.Compose(
-            [
-                transforms.Resize(size, interpolation=transforms.InterpolationMode.BILINEAR),
-                transforms.CenterCrop(size) if center_crop else transforms.RandomCrop(size),
-                transforms.ToTensor(),
-                transforms.Normalize([0.5], [0.5]),
-            ]
-        )
+        if multires_enabled:    
+            self.image_transforms = transforms.Compose(
```

Comment:
@patil-suraj what do you think regarding multires?

Reply:
Not really in favour of this. I think better to keep the script simpler.